doc/source/tutorial/signal.rst

```diff
@@ -905,86 +905,453 @@ in the amplitude response.
+   calculating the integral of the sinc function:
 
-   >>> f, Pwelch_spec = signal.welch(x, fs, scaling='spectrum')
+        In [1]: import sympy as sy
 
-   >>> plt.semilogy(f, Pwelch_spec)
-   >>> plt.xlabel('frequency [Hz]')
-   >>> plt.ylabel('PSD')
-   >>> plt.grid()
-   >>> plt.show()
+        In [2]: a = sy.Symbol('a')
+           ...: tau = sy.Symbol("tau", positive=True)
+           ...: f = sy.Symbol('f', real=True)
+
+        In [3]: Z = a * sy.sin(sy.pi*tau*f) / (sy.pi*f)
+
+        In [4]: sy.integrate(Z*Z.conjugate(), (f, -sy.oo, sy.oo))
+        Out[4]: a*tau*conjugate(a)
+
+as well. E.g., it can be shown by direct calculation that the energy of :math:`X(f)` of
+Eq. :math:numref:`eq_SpectA_FT_Energy` is :math:`|a|^2\tau/2`. Hence, the signal's power
+in a frequency band :math:`[f_a, f_b]` can be determined with
+
+.. math::
+    :label: eq_SpectA_FT_PowerI
+
+    P_X^{a,b} =  \frac{1}{\tau} \int_{f_a}^{f_b} \abs{X(f)}^2 \dd f\ .
+
+Thus the function :math:`|X(f)|^2` can be defined as the so-called "energy spectral
+density and :math:`S_{xx}(f) := |X(f)|^2 / \tau` as "power spectral density" (PSD) of
+:math:`x(t)`. Instead of the PSD, the so-called "amplitude spectral density"
+:math:`X(f) / \sqrt{\tau}` is also used, which still contains the phase information.
+Its absolute square is the PSD and thus it is closely related to the concept of the
+root-mean-square (RMS) value :math:`\sqrt{P_x}` of a signal.
+
+In summary, this subsection presented five ways to represent a spectrum:
+
+.. list-table:: Comparison of Spectral Representations of sine signal :math:`x(t)`
+                of Eq. :math:numref:`eq_SpectA_sine` with unit :math:`\text{V}`:
+   :header-rows: 1
+
+   * -
+     - Spectrum
+     - Amplitude Spectrum
+     - Energy Spectral Density
+     - Power Spectral Density (PSD)
+     - Amplitude Spectral Density
+   * - Definition:
+     - :math:`X(f)`
+     - :math:`X(f) / \tau`
+     - :math:`|X(f)|^2`
+     - :math:`|X(f)|^2 / \tau`
+     - :math:`X(f) / \sqrt{\tau}`
+   * - Magnitude at :math:`\pm f_x`:
+     - :math:`\frac{1}{2}|a|\tau`
+     - :math:`\frac{1}{2}|a|`
+     - :math:`\frac{1}{4}|a|^2\tau^2`
+     - :math:`\frac{1}{4}|a|^2\tau`
+     - :math:`\frac{1}{2}|a|\sqrt{\tau}`
+   * - Unit:
+     - :math:`\text{V} / \text{Hz}`
+     - :math:`\text{V}`
+     - :math:`\text{V}^2\text{s} / \text{Hz}`
+     - :math:`\text{V}^2 / \text{Hz}`
+     - :math:`\text{V} / \sqrt{\text{Hz}}`
+
+Note that the units presented in the table above are not unambiguous, e.g.,
+:math:`\text{V}^2\text{s} / \text{Hz} = \text{V}^2\text{s}^2 = \text{V}^2/
+\text{Hz}^2`. When using the absolute value of :math:`|X(f) / \tau|` of the amplitude
+spectrum, it is called magnitude spectrum. Furthermore, note that the naming scheme of
+the representations is not consistent and varies in literature.
+
+For real-valued signals the so-called "onesided" spectral is often utilized. It only
```

Comment:
```suggestion
For real-valued signals the so-called "onesided" spectrum is often utilized. It only
```

Reply:
I changed the wording to  `"onesided" spectral representation` in commit  27ad44bd7, since it is a little more precise than "spectrum" (probably knit-picking)